feconf.py

```diff
@@ -951,11 +952,19 @@ def get_empty_ratings():
 HANDLER_TYPE_HTML = 'html'
 HANDLER_TYPE_JSON = 'json'
 
-# Following are the constants for the roles.
+# Following are the constants for the roleIds. RoleIds are human
+# readable roles with underscores instead of spaces.
+# To check the role updation from /admin#roles, set ADMIN_SHOW_UPDATE_ROLE
+# constant in Admin.js to true.
 ROLE_GUEST = 'GUEST'
```

Comment:
I think it's fine as it is. Role Id is only in documentations. We don't have role id variable anywhere in code. 
So, it's your call. Should I change them?